hammr/commands/image/image.py

```diff
@@ -709,4 +732,13 @@ def deploy_azure(self, deploy_file, publish_image):
 
         deployed_instance_id = deployed_instance.applicationId
         status = show_deploy_progress_without_percentage(self, deployed_instance_id)
-        return print_deploy_info(self, status, deployed_instance_id)
\ No newline at end of file
+        return print_deploy_info(self, status, deployed_instance_id)
+
+    def get_all_images(self):
```

Comment:
I am not a big fan of this method.
When I call a "get_all" method, I expect it to return an empty list if there is no images (and not None), and I don't expect it to print something.
I understand it reduces the number of duplicated lines, but when I read code that call this method I haven't understand it will print "No images available" when there is no image available.
I prefer duplicating the print of the error message in that case.
Other ideas:
- rename the method ?
- If the case where there is no images is specific and should break the normal flow, then maybe it should raise an exception (which contains the message "No images available").
It will at least avoid to have to test if this method returns None or not.

Reply:
I have chosen here to return an empty list when there is no image. I did not raise the exception inside this method, because having no images is not an issue for certain commands (like image list).